app/assets/stylesheets/active_admin/_base.scss

```diff
@@ -4,38 +4,45 @@
 
 // Partials
 @import "./typography";
-@import "./header";
-@import "./forms";
-@import "./components/comments";
-@import "./components/flash_messages";
-@import "./components/date_picker";
-@import "./components/tables";
-@import "./components/batch_actions";
-@import "./components/modal_dialog";
-@import "./components/blank_slates";
-@import "./components/breadcrumbs";
-@import "./components/dropdown_menu";
-@import "./components/buttons";
-@import "./components/grid";
-@import "./components/links";
-@import "./components/pagination";
-@import "./components/panels";
-@import "./components/columns";
-@import "./components/scopes";
-@import "./components/status_tags";
-@import "./components/table_tools";
-@import "./components/index_list";
-@import "./components/unsupported_browser";
-@import "./components/tabs";
-@import "./pages/logged_out";
-@import "./structure/footer";
-@import "./structure/main_structure";
-@import "./structure/title_bar";
 
-body {
-  @include sans-family;
-  line-height: 1.5;
-  font-size: 72%;
-  background: $body-background-color;
-  color: $text-color;
+@media screen {
+  @import "./header";
+  @import "./forms";
+  @import "./components/comments";
+  @import "./components/flash_messages";
+  @import "./components/date_picker";
+  @import "./components/tables";
+  @import "./components/batch_actions";
+  @import "./components/modal_dialog";
+  @import "./components/blank_slates";
+  @import "./components/breadcrumbs";
+  @import "./components/dropdown_menu";
+  @import "./components/buttons";
+  @import "./components/grid";
+  @import "./components/links";
+  @import "./components/pagination";
+  @import "./components/panels";
+  @import "./components/columns";
+  @import "./components/scopes";
+  @import "./components/status_tags";
+  @import "./components/table_tools";
+  @import "./components/index_list";
+  @import "./components/unsupported_browser";
+  @import "./components/tabs";
+  @import "./pages/logged_out";
+  @import "./structure/footer";
+  @import "./structure/main_structure";
+  @import "./structure/title_bar";
+
+  body {
+    @include sans-family;
+    line-height: 1.5;
+    font-size: 72%;
+    background: $body-background-color;
+    color: $text-color;
+  }
+}
```

Comment:
The admin CSS is already loaded as a screen media stylesheet so this isn't necessary. It would be best to remove the screen media wrapper part.

```suggestion
@import "./header";
@import "./forms";
@import "./components/comments";
@import "./components/flash_messages";
@import "./components/date_picker";
@import "./components/tables";
@import "./components/batch_actions";
@import "./components/modal_dialog";
@import "./components/blank_slates";
@import "./components/breadcrumbs";
@import "./components/dropdown_menu";
@import "./components/buttons";
@import "./components/grid";
@import "./components/links";
@import "./components/pagination";
@import "./components/panels";
@import "./components/columns";
@import "./components/scopes";
@import "./components/status_tags";
@import "./components/table_tools";
@import "./components/index_list";
@import "./components/unsupported_browser";
@import "./components/tabs";
@import "./pages/logged_out";
@import "./structure/footer";
@import "./structure/main_structure";
@import "./structure/title_bar";

body {
  @include sans-family;
  line-height: 1.5;
  font-size: 72%;
  background: $body-background-color;
  color: $text-color;
}
```

Reply:
Actually, this is an unintentional mistake on my part, I'm loading the (now only) stylesheet with `media="screen"`, but the stylesheet should be loaded for all media types, right?

If I understood your idea correctly, these blocks are correct because otherwise all the styles will be applied in the "print view" which is unintented (at least not how it works currently).

Reply:
Yes that makes sense about the separate screen/print blocks because in #3862 the print styles were updated so many of the elements that were originally displayed in print mode weren't anymore. Thanks!